src/nebula-hooks/use-extended-theme.ts

```diff
@@ -44,12 +44,10 @@ export const tableThemeColors = (theme: ExtendedTheme, rootElement: HTMLElement)
 const useExtendedTheme = (rootElement: HTMLElement): ExtendedTheme => {
   // TODO: add name method to stardust.Theme
   const nebulaTheme = useTheme() as ExtendedTheme;
-  const theme = useMemo(
+  return useMemo(
```

Comment:
this works the same? never seen a useMemo been directly returned

Reply:
yes, typescript can help to verify that the type is the same. I will look into how to write unit tests for the custom hook.

Reply:
yeah I mean if there is a simple way to do it. We have a really high test coverage so more tests does not necessarily add more value

Reply:
Okay, it is not easy. then I will not do it.